aws-ts-static-website/index.ts

```diff
@@ -191,6 +231,8 @@ const distributionArgs: aws.cloudfront.DistributionArgs = {
         maxTtl: tenMinutes,
     },
 
+    orderedCacheBehaviors: cacheBehaviours,
```

Comment:
@clstokes, I think we talked about this offline but when I saw the code then my question was why the `cacheBehaviors` is attached to `orderedCacheBehaviors` here instead of added to the defaults. Perhaps mention your reasoning in a comment?